### ISSUE TITLE
doc: Update Contribution Section And Add Translation Section

### ISSUE THREAD

**Issue by PepperTheVixen** [stardew-access/stardew-access#335]
In the [contributors section of the readme](https://github.com/khanshoaib3/stardew-access?tab=readme-ov-file#contributors), there is an image with no alt text. I am not able to add it since I'm blind.

The link looks like it's using some auto generator. Could we find a solution that includes alt text for better visual accessibility?

**Comment by khanshoaib3** [comment]
@PepperTheVixen Yeah, I noticed that too. I'll replace contrib.rocks with something more accessible, I saw one being used in a repo that looked accessible, I don't know the tools name though.

I'll update the contribution section before release.
